src/servers/middlewares/auth-router.ts

```diff
@@ -38,9 +36,9 @@ authRouter.post("/browser", async (req, res) => {
   const logger = graphqlLogger.child({ ip, body: req.body })
 
   try {
-    const { data } = await kratos.toSession(undefined, req.header("Cookie"))
+    const { data } = await kratosPublic.toSession(undefined, req.header("Cookie"))
 
-    const kratosLoginResp = await Users.loginWithKratos({
+    const kratosLoginResp = await Users.loginWithEmail({
```

Comment:
I'm wondering if the boundary is still correct here. Perhaps this should actually be `loginWithCookie` and the use case calls the `AuthService` to resolve the cookie etc.

Reply:
I didn't really touch this code (expect a function rename). This code is meant to be deleted on Cookie is activated and webwallet updated. (it was a "hack"/proxy for when there was no Oathkeeper that knew how to manage cookie)